openff/utilities/_tests/test_provenance.py

```diff
@@ -2,20 +2,56 @@
 
 import pytest
 
-from openff.utilities.provenance import get_ambertools_version
+from openff.utilities.provenance import (
+    _get_conda_list_package_versions,
+    get_ambertools_version,
+)
+from openff.utilities.warnings import CondaExecutableNotFoundWarning
+
+
+def test_conda_unavailable_returns_empty_dict(monkeypatch):
+    """
+    Test that if `conda|etc. list` fails, with or without AmberTools (properly) installed,
+    get_ambertools_version returns `None`.
+    """
+
+    # TODO: Look into mocking with a context manager?
+    def mock_has_executable(executable_name: str) -> bool:
+        return False
+
+    import openff.utilities.utilities
+
+    monkeypatch.setattr(
+        openff.utilities.utilities,
+        "has_executable",
+        mock_has_executable,
+    )
+
+    with pytest.warns(
+        CondaExecutableNotFoundWarning,
+        match="executable not found",
+    ):
+        assert _get_conda_list_package_versions() == dict()
+
+    with pytest.warns(
+        CondaExecutableNotFoundWarning,
+        match="executable not found",
+    ):
+        assert get_ambertools_version() is None
 
 
-def test_get_ambertools_version_found():
-    # Skip if ambertools is not installed.
+def test_conda_available_get_ambertools_version_found():
+    # Skip if ambertools is not installed - sander a better import, if not broken
```

Comment:
```suggestion
    # Skip if ambertools is not installed - sander would a better import test 
    # for AmberTools (since parmed could come from the standalone 
    # parmed package), however it's currently broken on ARM macs.
```